iroha_data_model/src/lib.rs

```diff
@@ -2146,72 +2146,72 @@ pub mod pagination {
         #[test]
         fn empty() {
             assert_eq!(
-                vec![1, 2, 3]
+                vec![1_i32, 2_i32, 3_i32]
```

Comment:
Could we please just allow this lint? It is way annoying to type constants like this. Especially in tests which should be quick and dirty?

Reply:
I think it is okay to allow it in tests, though I only had to correct a few places. Most are ok as is right now.

Reply:
I think you can allow it in tests, when you need it.

Reply:
IMO we should generally disable it. It doesn't help reading code at all as you rarely want to know types of constants.